en/README.md

```diff
@@ -70,8 +70,11 @@ Clojure | [clj-mavlink](https://github.com/WickedShell/clj-mavlink) | Y | Y | Y
 
 The core development team and community are active on the following chat channel:
 
-* [Slack](http://slack.px4.io) (sign up) - #mavlink channel
 * [Mailing list](https://groups.google.com/forum/#!forum/mavlink)
+* [Slack](http://slack.px4.io) (sign up) - #mavlink channel
+* [Gitter](https://gitter.im/ArduPilot/pymavlink) - for mavgen and other pymavlink tools
+* [ArduPilot Forum](https://discuss.ardupilot.org/c/ground-control-software/mavlink)
```

Comment:
Is it worth clarifying that the forums (PX4 and ArduPilot) are for questions about integrating with specific autopilots? 

Maybe too obvious, but was thinking  ....
```
AutoPilot Forums - MAVLink support specific to a particular autopilot:
* [ArduPilot Forum](https://discuss.ardupilot.org/c/ground-control-software/mavlink)
* [PX4 Forum](https://discuss.px4.io/c/mavlink)
```

Reply:
I’m on the fence as to whether to include them or not actually. We don’t discuss specific implementations elsewhere, really.

Reply:
The reason I'm "for" this change is that the fact is that most parts of the spec are optional, and the ability to find out what is supported is very large-grained. It makes sense to include links to be able to ask "is `MAV_CMD` whatever supported. 

Is there a better way to do this?

Reply:
Let's settle on the MAVLink mailing list. If this is to be a standard splitting tools up between implementer communities is not going to work.